source/clear-containers/getting-started.rst

```diff
@@ -3,14 +3,10 @@
 Clear Containers getting started guide
 ######################################
 
-
-Introduction
-============
-
-The Intel® Clear Containers v2.1.5 release is now available.
-This release enables executing existing Docker applications in the secure
-and fast Intel Clear Containers environment under Docker\* 1.12 and beyond
-via an :abbr:`Open Container Initiative (OCI)` compatible `runtime`.
+The Intel® Clear Containers enable executing existing Docker applications in
+the secure and fast Intel Clear Containers environment under Docker\*
+v17.05.0-ce and beyond via an :abbr:`Open Container Initiative (OCI)`
```

Comment:
I agree, we can leave it like this for now and once we move to the new system the release version will update alongside the code.